browser.js

```diff
@@ -4,15 +4,16 @@ var readystate = module.exports = require('./readystate')
   , win = (new Function('return this'))()
   , complete = 'complete'
   , root = true
-  , doc = win.document
-  , html = doc.documentElement;
+  , doc = win ? win.document : null
+  , html = doc ? doc.documentElement : null
+  , docReadyState = doc ? doc.readyState : null;
```

Comment:
I think this is no ok. It's caching the value of `document.readyState` so we will always use the cached value even if it changes.

Reply:
ooops, sorry, you are right. fixed that by turning docReadState into a function